.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: Build gh-pages
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+
+# GITHUB_TOKEN permissions to allow deployment to GitHub Pages
+permissions:
```

Comment:
Do not need these permissions for the build stage

Reply:
Leaving it there, going to look at running dev deploy in the next sprint or two - causing too many issues